GoogleSignIn/Sources/GIDProfileData_Private.h

```diff
@@ -28,6 +36,9 @@ NS_ASSUME_NONNULL_BEGIN
                    familyName:(nullable NSString *)familyName
                      imageURL:(nullable NSURL *)imageURL;
```

Comment:
Mark this with `NS_DESIGNATED_INITIALIZER`.

Reply:
Done.